src/sentry/sentry_metrics/indexer/cloudspanner/cloudspanner.py

```diff
@@ -42,12 +82,21 @@ class RawCloudSpannerIndexer(StringIndexer):
     and the corresponding reverse lookup.
     """
 
-    def __init__(self, instance_id: str, database_id: str) -> None:
+    def __init__(
+        self,
+        instance_id: str,
+        database_id: str,
+        table_name: str,
+        unique_organization_string_index: str,
```

Comment:
@MeredithAnya , @lynnagara  how are we going to configure this ? 
For postgres the `Configuration` object references a model, which means a table. Then we have infrastructure that routes tables to postgres instances in getsentry. 
How are we going to do for Spanner? 
Will we have a key that references an instance in the `Configuration` object and that instance is configured in server.py with all its details ? Or will we have the details directly in the `Configuration` object ?

Reply:
I don't think I understand what this is. Why is it being provided by the user rather than being a hardcoded value (the index name we defined on the table)?

Reply:
If we don't specify the index while performing a read of columns which don't filter by primary key you get an error like the one shown below. It expects the filtering key to be `id` which is the primary key.

grpc._channel._MultiThreadedRendezvous: <_MultiThreadedRendezvous of RPC that terminated with:
	status = StatusCode.FAILED_PRECONDITION
	details = "Wrong number of key parts for perfstringindexer. Expected: 1. Got: ["100000","cePzCBSawy"]"
	debug_error_string = "{"created":"@1660666723.204512000","description":"Error received from peer ipv4:142.250.191.42:443","file":"src/core/lib/surface/call.cc","file_line":967,"grpc_message":"Wrong number of key parts for perfstringindexer. Expected: 1. Got: ["100000","cePzCBSawy"]","grpc_status":9}"
>

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/nikharsaxena/Library/Application Support/JetBrains/PyCharm2021.1/scratches/scratch_10.py", line 180, in <module>
    for row in results:
  File "/Users/nikharsaxena/workspace/new/sentry/.venv/lib/python3.8/site-packages/google/cloud/spanner_v1/streamed.py", line 145, in __iter__
    self._consume_next()
  File "/Users/nikharsaxena/workspace/new/sentry/.venv/lib/python3.8/site-packages/google/cloud/spanner_v1/streamed.py", line 117, in _consume_next
    response = next(self._response_iterator)
  File "/Users/nikharsaxena/workspace/new/sentry/.venv/lib/python3.8/site-packages/google/cloud/spanner_v1/snapshot.py", line 59, in _restart_on_unavailable
    iterator = method(request=request)
  File "/Users/nikharsaxena/workspace/new/sentry/.venv/lib/python3.8/site-packages/google/cloud/spanner_v1/services/spanner/client.py", line 1445, in streaming_read
    response = rpc(
  File "/Users/nikharsaxena/workspace/new/sentry/.venv/lib/python3.8/site-packages/google/api_core/gapic_v1/method.py", line 145, in __call__
    return wrapped_func(*args, **kwargs)
  File "/Users/nikharsaxena/workspace/new/sentry/.venv/lib/python3.8/site-packages/google/api_core/timeout.py", line 102, in func_with_timeout
    return func(*args, **kwargs)
  File "/Users/nikharsaxena/workspace/new/sentry/.venv/lib/python3.8/site-packages/google/api_core/grpc_helpers.py", line 164, in error_remapped_callable
    six.raise_from(exceptions.from_grpc_error(exc), exc)
  File "<string>", line 3, in raise_from
google.api_core.exceptions.FailedPrecondition: 400 Wrong number of key parts for perfstringindexer. Expected: 1. Got: ["100000","cePzCBSawy"]

Reply:
I think lyn is suggesting to hardcode the index name because presumably, like column names, they are values you'd never want to change. Is that actually true? I.e. are index names namespaced by the table name? If they aren't that'd change things.

Reply:
I can hardcode the table name and index name. The reason I kept them configurable is that I wasn't sure if we wanted to support multiple tables (perf and release health) using spanner. And then different tables can have different indexes. But I guess it becomes too flexible to be able to configure. I will revert to hardcoding those names.

Reply:
With the introduction of using `use_case_id`, it is hardcoded now over here https://github.com/getsentry/sentry/pull/37802/files#diff-20db5e0348082b65c8a663a9a546b60cb007ecef98ba5ec99b8690402d9acdc3R30-R36